app/controllers/adhoq/executions_controller.rb

```diff
@@ -29,5 +38,9 @@ def respond_report(report)
         send_data report.data, type: report.mime_type, filename: report.name, disposition: 'attachment'
       end
     end
+
+    def async_execution?
+      defined?(ActiveJob) && Adhoq.config.async_execution && !Adhoq.current_storage.is_a?(Adhoq::Storage::OnTheFly)
```

Comment:
hmm.
Is there a way to hide reference `defined?(ActiveJob)` from controller?

eg. behind` Adhoq.config`
```ruby
def Adhoq.config.async_executable?
  defined?(ActiveJob) && Adhoq.config.async_execution
end
```

How do you think about this?